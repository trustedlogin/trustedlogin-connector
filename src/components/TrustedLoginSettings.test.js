import { render } from "@testing-library/react";
import TrustedLoginSettings from "./TrustedLoginSettings";
import Provider, { testTeam } from "./TestProvider";
describe("TrustedLoginSettings", () => {
  it("renders & matches snapshot, with on-boarding completed", () => {
    const Wrapper = (props) => (
      <Provider {...props} hasOnboarded={true} initialTeams={[testTeam]} />
    );

    const { container } = render(<TrustedLoginSettings />, {
      wrapper: Wrapper,
    });
    expect(container).toMatchSnapshot();
  });

  it("renders & matches snapshot, with on-boarding NOT completed", () => {
    const Wrapper = (props) => <Provider {...props} hasOnboarded={false} />;
    const { container } = render(<TrustedLoginSettings />, {
      wrapper: Wrapper,
    });
    expect(container).toMatchSnapshot();
  });
});
