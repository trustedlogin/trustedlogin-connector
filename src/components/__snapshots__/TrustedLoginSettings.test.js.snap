// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrustedLoginSettings renders & matches snapshot, with on-boarding NOT completed 1`] = `
<div>
  <div
    class="h-full overflow-hidden"
  >
    <div
      class="h-full flex bg-white"
    >
      <div
        class="flex flex-col min-w-0 flex-1 overflow-hidden"
      >
        <div
          class="flex-1 relative z-0 flex overflow-hidden"
        >
          <div
            class="flex-1 relative z-0 overflow-y-auto focus:outline-none md:order-last"
            id="trustedlogin-onboarding-main"
          >
            <div
              class=" inset-0 py-6 px-4 sm:px-6 lg:px-8"
            >
              <div
                class="h-full"
              >
                <div
                  class="flex flex-col pt-12 w-full max-w-sm mx-auto"
                >
                  <svg
                    class="mx-auto"
                    fill="none"
                    height="56"
                    viewBox="0 0 56 56"
                    width="56"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <rect
                      fill="#00AADD"
                      height="48"
                      rx="24"
                      width="48"
                      x="4"
                      y="4"
                    />
                    <path
                      d="M28 31V33V31ZM22 37H34C34.5304 37 35.0391 36.7893 35.4142 36.4142C35.7893 36.0391 36 35.5304 36 35V29C36 28.4696 35.7893 27.9609 35.4142 27.5858C35.0391 27.2107 34.5304 27 34 27H22C21.4696 27 20.9609 27.2107 20.5858 27.5858C20.2107 27.9609 20 28.4696 20 29V35C20 35.5304 20.2107 36.0391 20.5858 36.4142C20.9609 36.7893 21.4696 37 22 37ZM32 27V23C32 21.9391 31.5786 20.9217 30.8284 20.1716C30.0783 19.4214 29.0609 19 28 19C26.9391 19 25.9217 19.4214 25.1716 20.1716C24.4214 20.9217 24 21.9391 24 23V27H32Z"
                      stroke="white"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                    <rect
                      height="48"
                      rx="24"
                      stroke="#CDEFF9"
                      stroke-width="8"
                      width="48"
                      x="4"
                      y="4"
                    />
                  </svg>
                  <div
                    class="max-w-sm mx-auto mb-8 justify-center text-center"
                  >
                    <h2
                      class="mt-4 text-2xl text-gray-900"
                    >
                      Create your first team
                    </h2>
                    <p
                      class="mt-2 mb-4 text-sm text-gray-500"
                    >
                      Each team supports a project in TrustedLogin.
                    </p>
                    <a
                      class="text-blue-tl text-sm"
                      href="https://app.trustedlogin.com/settings#/teams"
                      target="_blank"
                    >
                      Where can I find this info?
                      <span
                        class="screen-reader-text"
                      >
                        (Link opens in a new window)
                      </span>
                    </a>
                  </div>
                  <form
                    class="flex flex-1 flex-col space-y-6"
                  >
                    <div>
                      <label
                        class="block text-sm font-medium text-gray-700"
                        for="account_id"
                      >
                        Account ID
                      </label>
                      <div
                        class="mt-2 relative rounded-lg"
                      >
                        <input
                          class="block w-full pl-3 pr-10 py-2.5 sm:text-sm border border-gray-300 rounded-lg focus:outline-none focus:border-sky-500 focus:ring-1 ring-offset-2 focus:ring-sky-500"
                          id="account_id"
                          name="account_id"
                          required=""
                          type="number"
                          value=""
                        />
                      </div>
                    </div>
                    <div>
                      <label
                        class="block text-sm font-medium text-gray-700"
                        for="public_key"
                      >
                        Public Key
                      </label>
                      <div
                        class="mt-2 relative rounded-lg"
                      >
                        <input
                          class="block w-full pl-3 pr-10 py-2.5 sm:text-sm border border-gray-300 rounded-lg focus:outline-none focus:border-sky-500 focus:ring-1 ring-offset-2 focus:ring-sky-500"
                          id="public_key"
                          name="public_key"
                          required=""
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                    <div>
                      <label
                        class="block text-sm font-medium text-gray-700"
                        for="private_key"
                      >
                        Private Key
                      </label>
                      <div
                        class="mt-2 relative rounded-lg"
                      >
                        <input
                          class="block w-full pl-3 pr-10 py-2.5 sm:text-sm border border-gray-300 rounded-lg focus:outline-none focus:border-sky-500 focus:ring-1 ring-offset-2 focus:ring-sky-500"
                          id="private_key"
                          name="private_key"
                          required=""
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                    <div>
                      <label
                        class="block text-sm font-medium text-gray-700"
                        for="approved_roles"
                      >
                        What Roles Provide Support?
                      </label>
                      <div
                        class="mt-2"
                      >
                        <div
                          class="multiselect-container multiSelectContainer "
                          id="approved_roles"
                        >
                          <div
                            class="search-wrapper searchWrapper "
                          >
                            <input
                              autocomplete="off"
                              class="searchBox"
                              id="approved_roles_input"
                              placeholder="Select"
                              type="text"
                              value=""
                            />
                          </div>
                          <div
                            class="optionListContainer displayNone"
                          >
                            <ul
                              class="optionContainer"
                            >
                              <li
                                class="
              highlightOption highlight 
              false 
              false option
            "
                              >
                                Administrator
                              </li>
                              <li
                                class="
               
              false 
              false option
            "
                              >
                                Editor
                              </li>
                            </ul>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div>
                      <label
                        class="block text-sm font-medium text-gray-700"
                        for="helpdesk"
                      >
                        Help Desk
                      </label>
                      <div
                        class="mt-2"
                      >
                        <select
                          class="bg-white block w-full pl-3 pr-8 py-2.5 sm:text-sm border border-gray-300 rounded-lg focus:outline-none focus:border-sky-500 focus:ring-1 ring-offset-2 focus:ring-sky-500"
                          id="helpdesk"
                          name="helpdesk"
                        >
                          <option>
                            Select a Help Desk
                          </option>
                          <option
                            value="helpscout"
                          >
                            Help Scout
                          </option>
                          <option
                            value="freescout"
                          >
                            FreeScout
                          </option>
                        </select>
                      </div>
                    </div>
                    <div
                      class="pt-2"
                    >
                      <button
                        class="w-full inline-flex justify-center rounded-lg border border-transparent px-4 py-2.5 bg-blue-tl text-base font-medium text-white hover:bg-indigo-700 focus:outline-none focus:ring-2 ring-offset-2 focus:ring-sky-500 sm:col-start-2 sm:text-sm"
                        type="button"
                      >
                        Continue
                      </button>
                    </div>
                  </form>
                </div>
                <div
                  class="inline-flex pt-12 items-center justify-center sm:hidden"
                >
                  <a
                    class="text-sm text-blue-tl"
                    href="#"
                  >
                    Need Help? View our Documentation
                  </a>
                  <svg
                    class="ml-3"
                    fill="none"
                    height="14"
                    viewBox="0 0 14 14"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M1.16663 7.00008H12.8333M12.8333 7.00008L6.99996 1.16675M12.8333 7.00008L6.99996 12.8334"
                      stroke="#00AADD"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1.67"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <aside
            class="hidden relative bg-gray-tl md:order-first md:flex md:flex-col flex-shrink-0 min-w-[26rem] overflow-y-auto"
            id="trustedlogin-onboarding-side"
          >
            <div
              class="inset-0"
            >
              <div
                class="h-full flex flex-col justify-between p-12"
              >
                <div
                  class="space-y-16"
                >
                  <div>
                    <svg
                      class="trustedlogin-horizontal-logo mx-auto"
                      height="371.1px"
                      style="enable-background: new 0 0 1299.9 371.1;"
                      version="1.1"
                      viewBox="0 0 1299.9 371.1"
                      width="1299.9px"
                      x="0px"
                      xml:space="preserve"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink"
                      y="0px"
                    >
                      <g>
                        <g>
                          <g>
                            <path
                              class="tl-hl-st0"
                              d="M157.5,75.6c-26.5,0-48.1,21.6-48.1,48.1v39.8h20.6v-39.8c0-15.2,12.3-27.5,27.5-27.5 c15.1,0,27.5,12.3,27.5,27.5v39.8h20.6v-39.8C205.6,97.2,184.1,75.6,157.5,75.6z"
                            />
                            <g>
                              <path
                                class="tl-hl-st1"
                                d="M157.5,150.9c-38.5,0-69.7,4.8-69.7,10.8v79.8c0,38.5,47.5,54.8,69.7,54.8c22.2,0,69.7-16.3,69.7-54.8 v-79.8C227.2,155.7,196,150.9,157.5,150.9z"
                              />
                            </g>
                          </g>
                        </g>
                        <g>
                          <path
                            class="tl-hl-st2"
                            d="M157.5,171.1c-23,0-42.5,15.3-48.9,36.2h14.8c5.8-13.1,18.9-22.3,34.1-22.3c20.5,0,37.2,16.7,37.2,37.2 c0,20.5-16.7,37.2-37.2,37.2c-15.2,0-28.3-9.2-34.1-22.3h-14.8c6.4,20.9,25.9,36.2,48.9,36.2c28.2,0,51.1-22.9,51.1-51.1 C208.6,194,185.7,171.1,157.5,171.1z"
                          />
                        </g>
                        <path
                          class="tl-hl-st2"
                          d="M177.2,219.9l-28.3-16.4c-2.2-1.3-4-0.2-4,2.3v9.8H87.9v13h56.9v9.8c0,2.5,1.8,3.6,4,2.3l28.3-16.4 C179.3,223.2,179.3,221.2,177.2,219.9z"
                        />
                      </g>
                      <g>
                        <path
                          class="tl-hl-st1"
                          d="M298.5,183.3h-26.3v-18.2H346v18.2h-26.3v68.5h-21.2V183.3z"
                        />
                        <path
                          class="tl-hl-st1"
                          d="M357.8,165.1H388c8.8,0,13.1,0.6,16.8,2.1c9.8,3.8,16,12.6,16,25.2c0,9.2-4.4,19.3-13,23.6v0.2 c0,0,1.2,1.6,3.3,5.2l16.8,30.3h-23.6L389,222.5H379v29.2h-21.2V165.1z M388.2,204.4c6.8,0,11-3.8,11-10.7 c0-6.5-2.4-10.4-13.1-10.4H379v21.1H388.2z"
                        />
                        <path
                          class="tl-hl-st1"
                          d="M439.7,165.1H461v54.8c0,9.1,5.9,13.9,15,13.9c9.1,0,15-4.8,15-13.9v-54.8h21.2v54.8 c0,19.5-14.3,33.3-36.1,33.3c-22.1,0-36.4-13.8-36.4-33.3V165.1z"
                        />
                        <path
                          class="tl-hl-st1"
                          d="M537.3,224.7c0,0,9.8,9,20.7,9c4.4,0,9.1-1.7,9.1-6.9c0-10.3-39-9.9-39-36.6c0-16,13.4-26.5,30.7-26.5 c18.7,0,27.9,9.8,27.9,9.8l-9,17.3c0,0-8.7-7.6-19.1-7.6c-4.4,0-9.2,1.9-9.2,6.8c0,10.9,39,9,39,36.4c0,14.4-11,26.9-30.5,26.9 c-20.4,0-31.6-12.4-31.6-12.4L537.3,224.7z"
                        />
                        <path
                          class="tl-hl-st1"
                          d="M620.6,183.3h-26.3v-18.2h73.8v18.2h-26.3v68.5h-21.2V183.3z"
                        />
                        <path
                          class="tl-hl-st1"
                          d="M680,165.1h54.3v18.2h-33.1V199h26.4v18.2h-26.4v16.4H736v18.2h-56V165.1z"
                        />
                        <path
                          class="tl-hl-st1"
                          d="M752.5,165.1h30.8c27.3,0,44.7,15.9,44.7,43.1s-17.4,43.5-44.7,43.5h-30.8V165.1z M782.4,233.6 c14.7,0,23.6-8.7,23.6-25.3c0-16.5-9.3-25-23.6-25h-8.7v50.3H782.4z"
                        />
                        <path
                          class="tl-hl-st0"
                          d="M850.4,166.2h12v75.1h38.3v10.4h-50.3V166.2z"
                        />
                        <path
                          class="tl-hl-st0"
                          d="M950,164.7c24.7,0,43.9,19,43.9,43.6c0,25.2-19.1,44.8-43.9,44.8s-44-19.6-44-44.8 C906,183.8,925.3,164.7,950,164.7z M950,242.2c17.4,0,31.5-14.5,31.5-33.8c0-18.7-14.1-32.7-31.5-32.7s-31.6,14.1-31.6,32.7 C918.4,227.6,932.5,242.2,950,242.2z"
                        />
                        <path
                          class="tl-hl-st0"
                          d="M1050.4,164.7c20.5,0,31.1,10.3,31.1,10.3l-6.1,9c0,0-9.6-8.4-24.1-8.4c-20.1,0-32.2,14.7-32.2,33 c0,19.9,13.6,33.6,31.5,33.6c14.8,0,24.2-10.7,24.2-10.7v-12.4H1061v-10.4h24.8v43h-10.5v-5.3c0-1.8,0.1-3.6,0.1-3.6h-0.2 c0,0-9.3,10.4-26.3,10.4c-23,0-42-18.2-42-44.3C1006.8,184.1,1025.5,164.7,1050.4,164.7z"
                        />
                        <path
                          class="tl-hl-st0"
                          d="M1105.8,166.2h12v85.5h-12V166.2z"
                        />
                        <path
                          class="tl-hl-st0"
                          d="M1142.3,166.2h11.9l38,55.6c3.2,4.6,7.1,12.4,7.1,12.4h0.2c0,0-0.8-7.6-0.8-12.4v-55.6h12v85.5H1199 l-38.2-55.5c-3.2-4.7-7.1-12.5-7.1-12.5h-0.2c0,0,0.8,7.6,0.8,12.5v55.5h-12V166.2z"
                        />
                      </g>
                    </svg>
                  </div>
                  <div
                    style="visibility: hidden;"
                  >
                    <nav
                      aria-label="Progress"
                    >
                      <ol
                        class="overflow-hidden"
                        role="list"
                      >
                        <li
                          class="relative pb-10"
                        >
                          <div
                            aria-hidden="true"
                            class="-ml-px  mt-0.5 top-4 left-4 w-0.5 h-full bg-gray-300"
                          />
                          <a
                            class="relative flex items-start group"
                            href="#"
                          >
                            <span
                              class="h-9 flex items-center"
                            >
                              <span
                                class="relative z-10 w-8 h-8 flex items-center justify-center bg-white border-2 border-blue-tl rounded-full"
                              >
                                <span
                                  class="h-2.5 w-2.5 bg-blue-tl rounded-full"
                                />
                              </span>
                            </span>
                            <span
                              class="ml-4 min-w-0 flex flex-col"
                            >
                              <span
                                class="text-xs font-semibold tracking-wide uppercase text-blue-tl"
                              >
                                Link your account
                              </span>
                              <span
                                class="text-sm text-gray-500"
                              />
                            </span>
                          </a>
                        </li>
                        <li
                          class="relative pb-10"
                        >
                          <div
                            aria-hidden="true"
                            class="-ml-px  mt-0.5 top-4 left-4 w-0.5 h-full bg-gray-300"
                          />
                          <a
                            aria-current="step"
                            class="relative flex items-start group"
                            href="#"
                          >
                            <span
                              aria-hidden="true"
                              class="h-9 flex items-center"
                            >
                              <span
                                class="relative z-10 w-8 h-8 flex items-center justify-center bg-white border-2 border-gray-300 rounded-full group-hover:border-gray-400"
                              >
                                <span
                                  class="h-2.5 w-2.5 bg-transparent rounded-full group-hover:bg-gray-300"
                                />
                              </span>
                            </span>
                            <span
                              class="ml-4 min-w-0 flex flex-col"
                            >
                              <span
                                class="text-xs font-semibold tracking-wide uppercase text-gray-500"
                              >
                                Create Team
                              </span>
                              <span
                                class="text-sm text-gray-500"
                              />
                            </span>
                          </a>
                        </li>
                        <li
                          class="relative pb-10"
                        >
                          <div
                            aria-hidden="true"
                            class="-ml-px  mt-0.5 top-4 left-4 w-0.5 h-full bg-gray-300"
                          />
                          <a
                            aria-current="step"
                            class="relative flex items-start group"
                            href="#"
                          >
                            <span
                              aria-hidden="true"
                              class="h-9 flex items-center"
                            >
                              <span
                                class="relative z-10 w-8 h-8 flex items-center justify-center bg-white border-2 border-gray-300 rounded-full group-hover:border-gray-400"
                              >
                                <span
                                  class="h-2.5 w-2.5 bg-transparent rounded-full group-hover:bg-gray-300"
                                />
                              </span>
                            </span>
                            <span
                              class="ml-4 min-w-0 flex flex-col"
                            >
                              <span
                                class="text-xs font-semibold tracking-wide uppercase text-gray-500"
                              >
                                Configure Help Desk
                              </span>
                              <span
                                class="text-sm text-gray-500"
                              />
                            </span>
                          </a>
                        </li>
                      </ol>
                    </nav>
                  </div>
                </div>
                <div
                  class="inline-flex items-center justify-center"
                >
                  <a
                    class="text-sm text-blue-tl"
                    href="https://docs.trustedlogin.com/Vendor/intro"
                  >
                    Need Help? View our Documentation
                    <span
                      class="screen-reader-text"
                    >
                      (Link opens in a new window)
                    </span>
                  </a>
                  <svg
                    class="ml-3"
                    fill="none"
                    height="14"
                    viewBox="0 0 14 14"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M1.16663 7.00008H12.8333M12.8333 7.00008L6.99996 1.16675M12.8333 7.00008L6.99996 12.8334"
                      stroke="#00AADD"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1.67"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </aside>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`TrustedLoginSettings renders & matches snapshot, with on-boarding completed 1`] = `
<div>
  <div
    class="h-full overflow-hidden"
  >
    <header
      class="bg-white flex flex-1 auto justify-between items-center px-5 py-8 shadow-lg sm:px-10"
    >
      <div
        class=""
      >
        <svg
          class="trustedlogin-horizontal-logo mx-auto"
          height="371.1px"
          style="enable-background: new 0 0 1299.9 371.1;"
          version="1.1"
          viewBox="0 0 1299.9 371.1"
          width="1299.9px"
          x="0px"
          xml:space="preserve"
          xmlns="http://www.w3.org/2000/svg"
          xmlns:xlink="http://www.w3.org/1999/xlink"
          y="0px"
        >
          <g>
            <g>
              <g>
                <path
                  class="tl-hl-st0"
                  d="M157.5,75.6c-26.5,0-48.1,21.6-48.1,48.1v39.8h20.6v-39.8c0-15.2,12.3-27.5,27.5-27.5 c15.1,0,27.5,12.3,27.5,27.5v39.8h20.6v-39.8C205.6,97.2,184.1,75.6,157.5,75.6z"
                />
                <g>
                  <path
                    class="tl-hl-st1"
                    d="M157.5,150.9c-38.5,0-69.7,4.8-69.7,10.8v79.8c0,38.5,47.5,54.8,69.7,54.8c22.2,0,69.7-16.3,69.7-54.8 v-79.8C227.2,155.7,196,150.9,157.5,150.9z"
                  />
                </g>
              </g>
            </g>
            <g>
              <path
                class="tl-hl-st2"
                d="M157.5,171.1c-23,0-42.5,15.3-48.9,36.2h14.8c5.8-13.1,18.9-22.3,34.1-22.3c20.5,0,37.2,16.7,37.2,37.2 c0,20.5-16.7,37.2-37.2,37.2c-15.2,0-28.3-9.2-34.1-22.3h-14.8c6.4,20.9,25.9,36.2,48.9,36.2c28.2,0,51.1-22.9,51.1-51.1 C208.6,194,185.7,171.1,157.5,171.1z"
              />
            </g>
            <path
              class="tl-hl-st2"
              d="M177.2,219.9l-28.3-16.4c-2.2-1.3-4-0.2-4,2.3v9.8H87.9v13h56.9v9.8c0,2.5,1.8,3.6,4,2.3l28.3-16.4 C179.3,223.2,179.3,221.2,177.2,219.9z"
            />
          </g>
          <g>
            <path
              class="tl-hl-st1"
              d="M298.5,183.3h-26.3v-18.2H346v18.2h-26.3v68.5h-21.2V183.3z"
            />
            <path
              class="tl-hl-st1"
              d="M357.8,165.1H388c8.8,0,13.1,0.6,16.8,2.1c9.8,3.8,16,12.6,16,25.2c0,9.2-4.4,19.3-13,23.6v0.2 c0,0,1.2,1.6,3.3,5.2l16.8,30.3h-23.6L389,222.5H379v29.2h-21.2V165.1z M388.2,204.4c6.8,0,11-3.8,11-10.7 c0-6.5-2.4-10.4-13.1-10.4H379v21.1H388.2z"
            />
            <path
              class="tl-hl-st1"
              d="M439.7,165.1H461v54.8c0,9.1,5.9,13.9,15,13.9c9.1,0,15-4.8,15-13.9v-54.8h21.2v54.8 c0,19.5-14.3,33.3-36.1,33.3c-22.1,0-36.4-13.8-36.4-33.3V165.1z"
            />
            <path
              class="tl-hl-st1"
              d="M537.3,224.7c0,0,9.8,9,20.7,9c4.4,0,9.1-1.7,9.1-6.9c0-10.3-39-9.9-39-36.6c0-16,13.4-26.5,30.7-26.5 c18.7,0,27.9,9.8,27.9,9.8l-9,17.3c0,0-8.7-7.6-19.1-7.6c-4.4,0-9.2,1.9-9.2,6.8c0,10.9,39,9,39,36.4c0,14.4-11,26.9-30.5,26.9 c-20.4,0-31.6-12.4-31.6-12.4L537.3,224.7z"
            />
            <path
              class="tl-hl-st1"
              d="M620.6,183.3h-26.3v-18.2h73.8v18.2h-26.3v68.5h-21.2V183.3z"
            />
            <path
              class="tl-hl-st1"
              d="M680,165.1h54.3v18.2h-33.1V199h26.4v18.2h-26.4v16.4H736v18.2h-56V165.1z"
            />
            <path
              class="tl-hl-st1"
              d="M752.5,165.1h30.8c27.3,0,44.7,15.9,44.7,43.1s-17.4,43.5-44.7,43.5h-30.8V165.1z M782.4,233.6 c14.7,0,23.6-8.7,23.6-25.3c0-16.5-9.3-25-23.6-25h-8.7v50.3H782.4z"
            />
            <path
              class="tl-hl-st0"
              d="M850.4,166.2h12v75.1h38.3v10.4h-50.3V166.2z"
            />
            <path
              class="tl-hl-st0"
              d="M950,164.7c24.7,0,43.9,19,43.9,43.6c0,25.2-19.1,44.8-43.9,44.8s-44-19.6-44-44.8 C906,183.8,925.3,164.7,950,164.7z M950,242.2c17.4,0,31.5-14.5,31.5-33.8c0-18.7-14.1-32.7-31.5-32.7s-31.6,14.1-31.6,32.7 C918.4,227.6,932.5,242.2,950,242.2z"
            />
            <path
              class="tl-hl-st0"
              d="M1050.4,164.7c20.5,0,31.1,10.3,31.1,10.3l-6.1,9c0,0-9.6-8.4-24.1-8.4c-20.1,0-32.2,14.7-32.2,33 c0,19.9,13.6,33.6,31.5,33.6c14.8,0,24.2-10.7,24.2-10.7v-12.4H1061v-10.4h24.8v43h-10.5v-5.3c0-1.8,0.1-3.6,0.1-3.6h-0.2 c0,0-9.3,10.4-26.3,10.4c-23,0-42-18.2-42-44.3C1006.8,184.1,1025.5,164.7,1050.4,164.7z"
            />
            <path
              class="tl-hl-st0"
              d="M1105.8,166.2h12v85.5h-12V166.2z"
            />
            <path
              class="tl-hl-st0"
              d="M1142.3,166.2h11.9l38,55.6c3.2,4.6,7.1,12.4,7.1,12.4h0.2c0,0-0.8-7.6-0.8-12.4v-55.6h12v85.5H1199 l-38.2-55.5c-3.2-4.7-7.1-12.5-7.1-12.5h-0.2c0,0,0.8,7.6,0.8,12.5v55.5h-12V166.2z"
            />
          </g>
        </svg>
      </div>
      <div
        class="relative flex items-center space-x-2"
      >
        <div
          class="relative"
        >
          <button
            class="inline-flex items-center pl-2.5 pr-3 h-10 border border-gray-300 text-sm leading-4 rounded-lg text-gray-500 bg-white :outline-none ring-4 ring-blue-100"
          >
            <svg
              class="sm:mr-2"
              fill="none"
              height="20"
              viewBox="0 0 20 20"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M6 10C6 8.93913 6.42143 7.92172 7.17157 7.17157C7.92172 6.42143 8.93913 6 10 6L10 10L6 10Z"
                fill="#FFB000"
              />
              <circle
                cx="10"
                cy="10"
                r="8"
                stroke="#FFB000"
                stroke-width="2"
              />
            </svg>
            <span
              class="hidden sm:inline-flex"
            >
              <span
                class="mx-1 text-gray-900 font-medium"
              >
                0 of 1 Teams Connected
              </span>
            </span>
            <svg
              class="hidden sm:inline-flex ml-3"
              fill="none"
              height="6"
              viewBox="0 0 10 6"
              width="10"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M1 5L5 1L9 5"
                stroke="#1D2327"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-opacity="0.5"
                stroke-width="1.5"
              />
            </svg>
          </button>
        </div>
        <a
          class="inline-flex items-center px-3.5 h-10 border border-gray-300 text-sm leading-4 font-medium rounded-lg text-gray-900 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 ring-offset-2 focus:ring-sky-500"
          href="https://docs.trustedlogin.com/Vendor/intro"
        >
          <svg
            class="-ml-0.5 h-4 w-4 sm:mr-2"
            fill="none"
            height="14"
            viewBox="0 0 14 14"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M14 7C14 3.13 10.86 0 7 0C3.13 0 0 3.13 0 7C0 10.87 3.13 14 7 14C10.86 14 14 10.87 14 7ZM7.7 8.48H6.14V8.05C6.14 7.67 6.22 7.35 6.38 7.07C6.54 6.79 6.84 6.5 7.26 6.18C7.67 5.89 7.94 5.65 8.07 5.47C8.21 5.29 8.27 5.08 8.27 4.85C8.27 4.6 8.18 4.41 7.99 4.27C7.8 4.14 7.54 4.08 7.2 4.08C6.62 4.08 5.95 4.27 5.2 4.65L4.56 3.37C5.43 2.88 6.36 2.63 7.33 2.63C8.14 2.63 8.78 2.83 9.25 3.21C9.73 3.6 9.96 4.12 9.96 4.76C9.96 5.19 9.87 5.56 9.67 5.87C9.48 6.19 9.1 6.54 8.56 6.93C8.18 7.21 7.95 7.42 7.85 7.56C7.75 7.71 7.7 7.9 7.7 8.13V8.48ZM6.23 11.22C6.05 11.05 5.96 10.8 5.96 10.49C5.96 10.16 6.04 9.91 6.22 9.74C6.4 9.57 6.65 9.49 6.99 9.49C7.31 9.49 7.56 9.58 7.74 9.75C7.92 9.92 8.01 10.17 8.01 10.49C8.01 10.79 7.92 11.04 7.74 11.21C7.56 11.39 7.31 11.48 6.99 11.48C6.66 11.48 6.41 11.39 6.23 11.22Z"
              fill="currentColor"
            />
          </svg>
          <span
            class="hidden sm:inline-flex"
          >
            Need Help?
          </span>
        </a>
      </div>
    </header>
    <div
      class="flex flex-col px-5 py-6 sm:px-10"
    >
      <div
        class="pb-6 mb-6 border-b md:flex md:items-center md:justify-between md:space-x-5"
      >
        <div
          class="flex items-center space-x-5"
        >
          <div>
            <h2
              class="text-2xl text-gray-900"
            >
              Teams
            </h2>
            <p
              class="mt-1 text-sm text-gray-500"
            >
              Manage your TrustedLogin settings
            </p>
          </div>
        </div>
        <div
          class="mt-6 flex flex-col-reverse justify-stretch space-y-4 space-y-reverse sm:flex-row-reverse sm:justify-end sm:space-x-reverse sm:space-y-0 sm:space-x-3 md:mt-0 md:flex-row md:space-x-3"
        >
          <button
            class="ml-3 inline-flex leading-6 justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-lg text-white bg-blue-tl hover:bg-indigo-700 focus:outline-none focus:ring-2 ring-offset-2 focus:ring-sky-500"
            type="submit"
          >
            <svg
              class="mr-1"
              height="24"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
              >
                <path
                  d="M12 6v6m0 0v6m0-6h6m-6 0H6"
                  stroke="#FFFFFF"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </g>
            </svg>
            Add Team
          </button>
        </div>
      </div>
      <div
        class="flex flex-col justify-center w-full bg-white rounded-lg shadow"
      >
        <ul
          class="divide-y divide-gray-200 px-5 sm:px-8"
          role="list"
        >
          <li
            class="py-5 flex flex-col items-center justify-between sm:py-8 sm:flex-row"
          >
            <div
              class="flex w-full items-center space-x-5 sm:w-auto"
            >
              <button
                class="flex-shrink-0 flex items-center justify-center h-12 w-12 bg-purple-600 text-white text-sm font-medium rounded-lg"
              >
                TL
              </button>
              <div
                class="flex flex-row space-x-16 items-center w-full justify-between sm:justify-start"
              >
                <div
                  class="flex flex-col max-w-[10rem] sm:max-w-[8rem] md:max-w-none"
                >
                  <p
                    class="text-lg font-medium text-gray-900 leading-tight min-w-[6rem]"
                    id="team-option-1-label"
                  >
                    Team 0
                  </p>
                  <p
                    class="text-sm text-gray-500"
                    id="team-option-1-description"
                  >
                    #
                    1
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex items-center space-x-5 w-full mt-4 justify-between sm:w-auto sm:mt-0"
            >
              <div
                class="flex items-center space-x-6"
              >
                <button
                  class="text-sm text-blue-tl hover:text-navy-tl p-2"
                >
                  Edit
                </button>
                <button
                  class="text-sm text-red-500 hover:text-red-800 p-2"
                >
                  Delete
                </button>
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
